### PR TITLE
Support for XinaBox CC03 and CS11

### PR DESCRIPTION
**Pull request by KalbeAbbas**

### CODE REVIEW

**Comment by KalbeAbbas** [comment]
Ok. Thanks

**Comment by KalbeAbbas** [comment]
We are closing the PR and will come back after some upgrades. Thanks

File: _board/CC03.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: download
+board_id: "cc03"
```
Comment (ladyada) [inline _board/CC03.md]: needs to match https://github.com/adafruit/circuitpython/tree/master/ports/atmel-samd/boards/xinabox_cc03

File: _board/CS11.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+layout: download
+board_id: "cs11"
```
Comment (ladyada) [inline _board/CS11.md]: https://github.com/adafruit/circuitpython/tree/master/ports/atmel-samd/boards/xinabox_cs11

File: _board/CC03.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: download
-board_id: "cc03"
-title: "CC03 Download"
-name: "CC03"
+board_id: "XinaBox CC03
```
Comment (ladyada) [inline _board/CC03.md]: no, match the *folder name* https://github.com/adafruit/circuitpython/tree/master/ports/atmel-samd/boards/xinabox_cc03

File: _board/xinabox_cc03.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+layout: download
+board_id: "xinabox_cc03"
+title: "CC03 download"
+name: "XinaBox CC03"
+manufacturer: "XinaBox"
+board_url: "https://drive.google.com/file/d/13xfg0SRZEeO-_xMElqdf0Sne8rjAcfyn/view"
+board_image: "xinabox_cc03.jpg"
+features:
+  - ARM Cortex-M0+ Architecture
```
Comment (makermelissa) [inline _board/xinabox_cc03.md]: Please see the guide page at https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website/adding-to-downloads. On features, please only use the items in that list. If those are not features, either just leave it empty or remove the features block altogether. Thanks.

File: _board/xinabox_cs11.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+layout: download
+board_id: "xinabox_cs11"
+title: "CS11 Download"
+name: "XinaBox CS11"
+manufacturer: "XinaBox"
+board_url: "https://drive.google.com/file/d/1ZGZZyPdnp49XdEDqoT0ZOT0FdfJ4DM-z/view"
+board_image: "xinabox_cs11.jpg"
+features:
+  - ARM Cortex-M0+ Architecture
```
Comment (makermelissa) [inline _board/xinabox_cs11.md]: Same with this features list.

File: _board/xinabox_cc03.md (hunk 4 above)
Comment (KalbeAbbas) [reply _board/xinabox_cc03.md]: Thanks @makermelissa, I've updated .mds and uploaded new images.

**Pull request merged**